integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/webhooks/UrlPusher.java

```diff
@@ -0,0 +1,32 @@
+package no.difi.meldingsutveksling.webhooks;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.*;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.web.client.ResourceAccessException;
+import org.springframework.web.client.RestTemplate;
+
+@Slf4j
+@RequiredArgsConstructor
+public class UrlPusher {
+
+    private final RestTemplate restTemplate;
+
+    @Async("threadPoolTaskScheduler")
+    public void pushAsync(String uri, String jsonPayload) {
+        push(uri, jsonPayload);
+    }
+
+    void push(String uri, String jsonPayload) {
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON_UTF8);
+        log.debug("Pushing to {}", uri);
+        try {
+            ResponseEntity<String> responseEntity = restTemplate.exchange(uri, HttpMethod.POST, new HttpEntity<>(jsonPayload, headers), String.class);
+            log.debug("Response was {} {}", responseEntity.getStatusCode().value(), responseEntity.getStatusCode().getReasonPhrase());
+        } catch (ResourceAccessException e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)